src/CoreWCF.Http/src/CoreWCF/Channels/AspNetCoreReplyChannel.cs

```diff
@@ -97,8 +97,7 @@ internal async Task HandleRequest(HttpContext context)
 
             var requestContext = HttpRequestContext.CreateContext(_httpSettings, context);
             var httpInput = requestContext.GetHttpInput(true);
-            Exception requestException;
-            Message requestMessage = httpInput.ParseIncomingMessage(out requestException);
+            (Message requestMessage, Exception requestException) = await httpInput.ParseIncomingMessageAsync();
             requestMessage.Properties.Add("Microsoft.AspNetCore.Http.HttpContext", context);
```

Comment:
I think this can throw a NRE if the method returns an exception and null request

Reply:
You are right. Although that wasn't introduced by this change. I'll fix this asap.